### PR TITLE
Create 20190802-馒头虾球

### PR DESCRIPTION
**Pull request by immoonancient**
#7

### CODE REVIEW

File: subtitles/wang-gang/20190802-馒头虾球 [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Fried Prawn Balls with Steamed Buns
+
+首先我们把大虾去头
+First, remove the head of the prawn
+
+从背部切开
+and cut in the back
+
+去除虾线和虾壳
+Remove the prawn line and shell
```
Comment (NoraZheng) [inline subtitles/wang-gang/20190802-馒头虾球]: Peel and devein the prawns

File: subtitles/wang-gang/20190802-馒头虾球 [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+for 10 minutes
+
+下一步我们准备一个白馒头
+Next, prepare a white steamed bun
+
+将馒头切成大小均匀的小粒备用
+Cut the bun into even sized small dices
+
+然后将腌制好的大虾先沾上淀粉再裹上蛋液
+Coat the prawns first with starch, and then with egg liquid
```
Comment (NoraZheng) [inline subtitles/wang-gang/20190802-馒头虾球]: egg wash

File: subtitles/wang-gang/20190802-馒头虾球 [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+and add in a moderate amount of ketchup
+
+翻炒均匀之后加入白糖两勺
+Stir fry it, and then add in two spoons of white sugar
+
+再加入少许白醋
+and a little white vinegar
+
+然后加入适量的水淀粉收紧汤汁
+Then add some starch water to thicken the sauce
```
Comment (NoraZheng) [inline subtitles/wang-gang/20190802-馒头虾球]: 水淀粉= (starch) slurry

**Comment by immoonancient** [comment]
谢谢审核，非常专业！

**Pull request merged**